dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java

```diff
@@ -1733,23 +1730,28 @@ private FormDataMultiPart createFormMultiPart(final ContentType contentType, fin
         return formDataMultiPart;
     }
 
-    private void checkBrandNewContentlet(String fieldNameTitle, String fieldNameFile1, String fieldNameFile2, String inputFile1Text, String inputFile2Text, Contentlet brandNewContentlet) throws IOException {
+    private void checkBrandNewContentlet(String fieldNameTitle, String fieldNameFile1, String fieldNameFile2, Contentlet brandNewContentlet) throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'brandNewContentlet' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)